the-collab-lab/tcl-25-smart-shopping-list
# Home View, empty List view, Add view styling

## Description

This PR adds styling to the app's Home View, Add View, and empty List View

## Acceptance Criteria

- [x] App logo icon at the top of the Home View
- [x] Welcome message in the Home View
- [x] Nicely styled button to create a new shopping list  in the Home View
- [x] Nicely styled form to join an existing list  in the Home View
- [x] Styling for app nav links
- [x] Description prompting users to add an item in the List View
- [x] Nicely styled dialog in the List View
- [x] Nicely styled add button in the Add View
- [x] Nicely styled form for adding items in the Add View




## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

* From your terminal, pull down this branch with `git pull origin ku-ci-app-styling` and check that branch out with `git checkout ku-ci-app-styling` 
* Then `npm start` to launch the app
* Check that the Home View has a logo and a welcome message at the top of the screen
* Check styling of create-list and join-list buttons 
* Check styling of app navigation
* Check styling of a description prompting users to add an item in the List View
* Check styling of the form and add-button in the Add View
* Check styling of the dialog in the List View
